src/index.ts

```diff
@@ -68,21 +68,22 @@ const githubApi = new MyOctokit({
   },
   auth: 'token ' + settings.github.token,
   throttle: {
-    onRateLimit: async (retryAfter, options) => {
+    onRateLimit: async (retryAfter, options, octokit, retryCount) => {
+      console.log(options);
       console.log(
         `Request quota exhausted for request ${options.method} ${options.url}`
       );
       console.log(`Retrying after ${retryAfter} seconds!`);
       return true;
     },
-    onAbuseLimit: async (retryAfter, options) => {
+    onSecondaryRateLimit: async (retryAfter, options, octokit, retryCount) => {
       console.log(
-        `Abuse detected for request ${options.method} ${options.url}`
-      );
+        `SecondaryRateLimit detected for request ${options.method} ${options.url}`
+      )
       console.log(`Retrying after ${retryAfter} seconds!`);
       return true;
     },
-    minimumAbuseRetryAfter: 1000,
+    fallbackSecondaryRateRetryAfter: 120,
```

Comment:
It always takes several rounds of waiting 120s so it could be higher by default.

```suggestion
    fallbackSecondaryRateRetryAfter: 1200,
```

Reply:
For me it only happens when migrating MRs with a lot of comments/discussions. If I remember correctly it takes about 20 to 30 minutes to run again. 5 minutes is a good balance I think since it will retry until it works. If it's 30 and we retry after 20 minutes it will wait 40 minutes in total.